konlpy/jvm.py

```diff
@@ -12,10 +12,11 @@
 from konlpy import utils
 
 
-def init_jvm(jvmpath=None):
+def init_jvm(jvmpath=None, max_heap_size=1024):
     """Initializes the Java virtual machine (JVM).
 
     :param jvmpath: The path of the JVM. If left empty, inferred by :py:func:`jpype.getDefaultJVMPath`.
+    :param max_heap_size: Maximum memory usage limitation (Megabyte). Default is 1024 (1Gb). If you set this value too small, you may got out of memory. We recommend that you set it 1024 ~ 2048 or more at least. However, if this value is too large, you may see inefficient memory usage.
```

Comment:
`Default is 1024 (1Gb)` -> `Default is 1024 (1GB)`

`Gb` is normally the symbol for Giga*bits*.